src/main/java/Duke.java

```diff
@@ -1,10 +1,135 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Duke {
+    public static int MAX_TASK = 100;
```

Comment:
Hi, I think you could consider making this a final variable! Other than that I like how you named your variable name as MAX_TASK.